INZFS.MVC/Models/DynamicForm/CurrencyInputModel.cs

```diff
@@ -18,6 +18,22 @@ protected override IEnumerable<ValidationResult> ExtendedValidation(ValidationCo
                 {
                     yield return new ValidationResult(ErrorMessage, new[] { nameof(DataInput) });
                 }
+                else
+                {
+                    if (DataInput.Length > CurrentPage.MaxLength)
+                    {
+                        yield return new ValidationResult($"{CurrentPage.FriendlyFieldName} must be {CurrentPage.MaxLength} characters or fewer", new[] { nameof(DataInput) });
+                    }
+                    else
+                    {
+                        var currencyValue = System.Convert.ToInt64(DataInput);
+
+                        if (CurrentPage.FieldName.Equals("parent-recent-turnover") && currencyValue > 1500000)
```

Comment:
I think, this is ok for now. Probably, we should probably use things like CustomValidators and configure them in json file